src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/MultiSigWidget/partials/FinalizeStep/FinalizeStep.tsx

```diff
@@ -221,6 +224,16 @@ const FinalizeStep: FC<FinalizeStepProps> = ({
     }
   }, [isMultiSigExecuted, isMultiSigRejected, action]);
 
+  useEffect(() => {
+    if (isMultiSigExecuted && previousIsMultiSigExecuted === false) {
+      /**
+       * We need to remove all getDomainBalance queries once a payment or funding has been successfully completed
```

Comment:
Same here as with motions, should we add this check to the `useFinalizeSuccessCallback` hook and check for the correct action type so we don't remove the cache for each finalized multisig?
![image](https://github.com/user-attachments/assets/a4b44c62-86e2-40af-988c-35d702c1963c)